src/Datasource/EntityTrait.php

```diff
@@ -612,7 +610,7 @@ public function offsetSet($offset, $value): void
      * @param string $offset The offset to remove.
      * @return void
      */
-    public function offsetUnset($offset): void
+    public function offsetUnset(mixed $offset): void
```

Comment:
Shouldn't the offset be a scalar value?

Reply:
Have to be compatible with ArrayAccess. All of these implementations use mixed.

https://www.php.net/manual/en/class.arrayaccess.php